mailserver/mailserver_test.go

```diff
@@ -300,17 +320,11 @@ func (s *MailserverSuite) TestBloomFromReceivedMessage() {
 
 func (s *MailserverSuite) setupServer(server *WMailServer) {
 	const password = "password_for_this_test"
-	const dbPath = "whisper-server-test"
-
-	dir, err := ioutil.TempDir("", dbPath)
-	if err != nil {
-		s.T().Fatal(err)
-	}
 
 	s.shh = whisper.New(&whisper.DefaultConfig)
 	s.shh.RegisterServer(server)
 
-	err = server.Init(s.shh, &params.WhisperConfig{DataDir: dir, Password: password, MinimumPoW: powRequirement})
+	err := server.Init(s.shh, &params.WhisperConfig{DataDir: s.dataDir, Password: password, MinimumPoW: powRequirement})
```

Comment:
Honest question: why do we redeclare the `err` variable here?

Reply:
We don't That's the first declaration. The previous one was removed in this PR.